test/functional/signature/corrupted-certificate.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -n test-bundle -f /test-file "$TEST_NAME"
+	run sudo sh -c "echo invalid > $TEST_NAME/cert"
```

Comment:
misuse of run again. Also testlib has the function `write_to_protected_file [-a] <file> <stream>` that can be used to write to a sudo owned file instead.